tests/vmcheck/overlay.sh

```diff
@@ -45,6 +45,13 @@ rsync -rlv $INSTTREE/usr/ vmcheck/usr/
 if [ -d $INSTTREE/etc ]; then # on CentOS, the dbus service file is in /usr
   rsync -rlv $INSTTREE/etc/ vmcheck/usr/etc/
 fi
+
+# ✀✀✀ BEGIN hack to get --selinux-policy (https://github.com/ostreedev/ostree/pull/1114) ✀✀✀
+mount -o rw,remount /usr
+rsync -rlv vmcheck/usr/ /usr/
+# ✀✀✀ END hack to get --selinux-policy ✀✀✀
+
 ostree refs --delete vmcheck || true
-ostree commit -b vmcheck -s '' --tree=dir=vmcheck --link-checkout-speedup
+ostree commit -b vmcheck --selinux-policy vmcheck -s '' \
```

Comment:
Let's say `--selinux-policy=vmcheck` to be really clear we're pulling the policy from there?

Reply:
(And while we're here drop `-s ''` since it's not required nowadays?)